onboarding/source/amb_light_service.c

```diff
@@ -18,22 +26,57 @@ static void lightServiceTask(void * pvParameters);
 uint8_t initLightService(void) {
     /* USER CODE BEGIN */
     // Create the task and queue here.
+    BaseType_t xReturned = pdFAIL;
+    if (lightServiceTaskHandle == NULL){
+        xReturned = xTaskCreate(lightServiceTask,
+                                LIGHT_SERVICE_NAME,
+                                LIGHT_SERVICE_STACK_SIZE,
+                                NULL,
+                                LIGHT_SERVICE_PRIORITY,
+                                &lightServiceTaskHandle);
+    }
+    if(xLightServiceQueue == NULL){
+        xLightServiceQueue = xQueueCreate(LIGHT_SERVICE_QUEUE_LENGTH,
```

Comment:
If we failed to create the task, there's no point in creating the queue. Check `xReturned` before creating the queue.